raiden/network/proxies/token_network.py

```diff
@@ -885,6 +903,10 @@ def _close_preconditions(
             partner: Address,
             block_identifier: BlockSpecification,
     ):
+        if not self.client.can_query_state_for_block(block_identifier):
+            # At this point in time we can't check the preconditions due to pruned blocks
+            return
```

Comment:
Should this raise an exception that is caught in the caller where the caller would log something about this? as Augusto commented, this counts as a silent failure somehow.